packages/communication-react/review/communication-react.api.md

```diff
@@ -137,8 +137,7 @@ export type AzureCommunicationMeetingAdapterArgs = {
     userId: CommunicationUserIdentifier;
     displayName: string;
     credential: CommunicationTokenCredential;
-    chatThreadId: string;
-    callLocator: TeamsMeetingLinkLocator | GroupCallLocator;
+    meetingArgs: CallAndChatMeetingArgs | TeamsMeetingLinkLocator;
```

Comment:
we say meetingArgs however we acknowledge we dont want to call it "Meeting" and then we use terms like "CallAndChatMeeting"args. Do we feel this makes sense? 

I think calling has probably had this conversation of Coordinates/Locators, etc. I am curious what they have found here.

Also a thought is can someone repurpose a composite for multiple different calls?

Reply:
When we do the naming rechange this will like change from `meeting<Suffix>` to something else. I didn't want to put too much thought into it as we haven't firmly decided on the name yet.

Composites will hopefully be used for multiple calls in the future with a single adapter -- but the arguments going into the ACSAdapters are very geared towards the single meeting/call/chat you are planning to join -- this PR follows the same design to keep consistent.

When we do support incoming calls, multiple calls, possibly multiple chat threads we'll likely need a new adapter interface as accepting a single locator at the adapter creation time likely doesn't make sense. We have non-breaking options for this though such as creating a AzureCommunicationMultiCallAdapter or an interface like `switchCall(newlocator)`